com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -2078,7 +2077,6 @@ internal void SynchronizeNetworkObjects(ulong clientId)
                     {
                         continue;
                     }
-
```

Comment:
this is stupid to debate but I really don't like code on the line after `}`

Reply:
Well I would rather it be "exactly" the same as upstream. So prefer to keep the diff smaller so we don't have conflicts if we have to maintain fork.